### ISSUE TITLE
How can i use keyboard?

### ISSUE THREAD

**Issue by davidebettio** [mattzollinhofer/vue-typeahead-bootstrap#39]
I read that now is enabled the keyboard navigation, but I cannot use arrow keys to navigate the results.

**Comment by mazenchami** [comment]
I can use the keyboard navigation but is there a way to highlight the row to show the user is in focus? that doesn't seem to work for me

**Comment by BozoJim** [comment]
Please take a look at the examples in the docs. Both of the issues the two of you mentioned appear to be fine in the the docs examples. Maybe there's a subtle difference in how you're using the typeahead.
Things like initializing a field to `''` vs `null` have gotten me in the past.

**Comment by mattzollinhofer** [comment]
There are updated docs here: https://mattzollinhofer.github.io/vue-typeahead-bootstrap-docs/examples/examples.html#basic. Hopefully they help.

@mazenchami: The highlighting is provided by bootstrap and any custom styles (if you have any) in your project. This component doesn't provide much in the way of styling, it's only trying to integrate with bootstrap.

**Comment by mazenchami** [comment]
@mattzollinhofer understood. i realized i asked my question wrong, but ended up figuring it out.
i'm no longer using bootstrap, which might be why my issue popped up. i needed to add the `:focus` to my class with a `background-color: xyz` to show what item was highlighted.

**Comment by nguyenk** [comment]
@mattzollinhofer, please let me start by thanking you for this work, and for maintaining a package that wasn't yours.

I'm sorry to insist, but I really dont see any selection changing by arrow-keys in this example :  https://mattzollinhofer.github.io/vue-typeahead-bootstrap-docs/examples/examples.html#basic.

I'm surely missing something, but I cant figure out what...

**Comment by mattzollinhofer** [comment]
Well shoot, it looks like we broke arrow keys with a recent change. I'm not exactly sure which one, but I think if you [rollback to v2.4.0](https://www.npmjs.com/package/vue-typeahead-bootstrap/v/2.4.0) it'll work. Sorry about that 😞.

**Comment by nguyenk** [comment]
> Well shoot, it looks like we broke arrow keys with a recent change. I'm not exactly sure which one, but I think if you [rollback to v2.4.0](https://www.npmjs.com/package/vue-typeahead-bootstrap/v/2.4.0) it'll work. Sorry about that .

Thank you very much !
